crates/pulsar-core/src/pdk/process_tracker.rs

```diff
@@ -194,6 +213,21 @@ impl ProcessTracker {
                     }
                 }
             }
+            TrackerRequest::IsDescendantOf(descendant_request) => {
+                let r = self.is_descendant_of(descendant_request.pid, &descendant_request.image);
+                match r {
+                    Err(TrackerError::ProcessNotFound) => {
+                        log::warn!(
+                            "Error in the descendant request for {} with image {}",
+                            descendant_request.pid,
+                            descendant_request.image
+                        );
```

Comment:
This would crash pulsar because we're not sending any response over `tx_reply` (and we have a `rx_reply.await.unwrap()` above.
I would log the error and add just reply `false` in these cases.

Reply:
..or we could reply `Err(ProcessNotFound)` and let the calling code decide what to do (likely to treat it a a false reply). Note that currently `is_descendant_of` returns Result which can only be Ok, never Err